webapp/channels/src/components/widgets/icons/fa_warning_icon.tsx

```diff
@@ -2,27 +2,27 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-
-import LocalizedIcon from 'components/localized_icon';
-
-import {t} from 'utils/i18n';
+import {injectIntl, type IntlShape} from 'react-intl';
 
 type Props = {
-    additionalClassName: string | null;
+    additionalClassName?: string;
+    intl: IntlShape;
 }
 
-export default class WarningIcon extends React.PureComponent<Props> {
-    public static defaultProps: Props = {
+class WarningIcon extends React.PureComponent<Props> {
+    public static defaultProps: Partial<Props> = {
         additionalClassName: null,
     };
 
     public render(): JSX.Element {
         const className = 'fa fa-warning' + (this.props.additionalClassName ? ' ' + this.props.additionalClassName : '');
```

Comment:
we can simplify here by using the `classname('fa fa-warning', this.props.additionalclass)`
the library will take care of the string concatenations and empty string checks. That way we don't have to add null as default value